packages/types/v1/surveys.ts

```diff
@@ -239,6 +239,10 @@ export const ZSurvey = z.object({
   autoComplete: z.union([z.number(), z.null()]),
   closeOnDate: z.date().nullable(),
   surveyClosedMessage: ZSurveyClosedMessage,
+  brandColor: z.union([z.string(), z.null()]),
```

Comment:
please use `.nullable()` instead of union as it improves readability:
```
brandColor: z.string().nullable(),
```

Reply:
We also have a regex validation we already use in other colors:

```ts
 brandColor: z.string().regex(/^#([A-Fa-f0-9]{6}|[A-Fa-f0-9]{3})$/).nullable(),
```

Reply:
sure will make those changes